src/test/java/org/opensearch/ubi/UbIParametersTests.java

```diff
@@ -80,7 +80,7 @@ public void reset() throws IOException {
     }
 
     public void testUbiParameters() {
-        final UbiParameters params = new UbiParameters("query_id", "user_query", "client_id", "object_id", Collections.emptyMap());
+        final UbiParameters params = new UbiParameters("query_id", "user_query", "client_id", "app", "object_id", Collections.emptyMap());
```

Comment:
`app` or `application`?    I think, if these are the params that are specified in the UBI Spec that it would be `application`???

Reply:
These are test values. I should make that obvious, i.e. make `query_id` a UUID.